config/application.rb

```diff
@@ -18,4 +18,28 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :delayed_job
   end
+
```

Comment:
I didn't really know where to put this common concern of config. It's really only needed in the `production.rb` but I wanted to test it out in development mode too. Putting these things in an initialiser doesn't seem to work for me, but maybe it would be better?